src/domain/collaboration/callout/CalloutTemplateCard/CalloutTemplateCard.tsx

```diff
@@ -0,0 +1,40 @@
+import { Skeleton } from '@mui/material';
```

Comment:
this one may need to be removed, it's what we talked, CalloutTemplateCard, `domain/collaboration/post/PostTemplateCard/PostTemplateCard.tsx` , `domain/collaboration/whiteboard/WhiteboardTemplateCard/WhiteboardTemplateCard.tsx`  and `domain/collaboration/InnovationFlow/InnovationFlowTemplateCard/InnovationFlowTemplateCard.tsx`
are using old cards